<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TapInfo</class>
 <widget class="QDialog" name="TapInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>279</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>279</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tap driver Info</string>
  </property>
  <property name="windowIcon">
   <iconset resource="systray.qrc">
    <normaloff>:/images/appicon.png</normaloff>:/images/appicon.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>261</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>TAP-Win32 Adapter:</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Incomming packets total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>51</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Outgoing packets total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIncommingTotal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>31</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutgoingTotal">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Incomming packets per second:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Outgoing packets per second:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIncommingPerSecond">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>83</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutgoingPerSecond">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>101</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="cmdClose">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>159</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="systray.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>16</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="systray.qrc"/>
 </resources>
 <connections/>
</ui>
